# **AE 330 Assignment 3**

**V S HARIKRISHNA**

**ROLL NO 160010054**

# Problem statement

Consider a mission where we have to launch a satellite/module such that it can dock with the International Space Station (ISS). ISS is orbiting in a low earth orbit 400 km above the earth’s surface, with an orbital speed of 7.66 km/s. Let this module mass be 15 tons, and the maximum acceleration that it can sustain is 4g.

Since we would eventually like to build our own space station, you have been asked to design a launch vehicle to put this 15 ton payload in the ISS orbit.

The manufacturer of the Vikas Engine has offered to provide an N2O4+UDMH engine with a characteristic velocity and structural coefficient matching those of the Vikas Engine. They are further offering flexibility in the nozzle area ratio, propellant mass, and burn time. The booster version of this engine is known to provide a specific impulse of 300 s. Upper-stage versions can have higher Isp if we use higher area ratios.

Similarly, Liquid Propulsion Systems Complex (LPSC) is offering LOX+LH2 engines with a characteristic velocity and structural coefficient of their cryogenic engine (CUS series), with flexibility in the nozzle area ratios and also propellant mass and burn times.



Further, Vikram Sarabhai Space Center (VSSC) is also ready to supply the S200 solid rocket motors in the same form as they are used for GSLV Mk.III.

Let the chamber pressure, propellant combination, etc. be the same as the engines being used on GSLV. Assume that you have full flexibility in the choice of nozzle parameters as per your flight environment.

Obtain the necessary information for these engines from the data available online (ISRO pages, wikipedia, astronautix etc.)

Design the launch vehicle. How many stages would be appropriate? Which engines will you use in these stages? Justify your choices. For each stage, give the mass breakup (propellant, structure, payload). What are the engine performance parameters (Isp, thrust coefficient, etc.)? At the end of your design, verify that your Delta V and acceleration are as per the requirements.

Submit your analysis as a report in pdf form.

Make reasonable assumptions wherever necessary.

# Rocket Stage Information

## Vikas Engine (PSLV Stage 2)

The second stage of the PSLV rocket is 2.8 meters in diameter, 12.8 meters long and has a liftoff mass of 46,000 Kilograms with an empty mass of 5,300 Kilograms.

The stage uses Unsymmetrical Di-Methylhydrazine / Hydrazine Hydrate as fuel and Nitrogen Tetroxide as oxidizer. It is powered by a single 799-Kilonewton Vikas engine. Vikas was developed by ISRO in the 1970s.

The engine has a dry weight of 900 Kilograms and operates at a chamber pressure of 58.5 bar and provides a specific impulse of 294 sec.

The second stage burns for 158 seconds. Vehicle control during second stage flight is accomplished by gimbaling the main engine by up to 4 degrees, roll control is provided by a Hot Gas Reaction Control Motor.

Second stage separation is accomplished with a merman band system and separation motors. It occurs at an altitude of 277 Kilometers.

 <center><img src="https://drive.google.com/uc?export=view&id=1iaotSbz_SVjjrZw8dv2OJUoAhc2ByiVu" alt="drawing" width="320"/></center>


<br>
<br>
<br>
<br>


## Cryogenic Upper Stage C-25 (GSLV Mk III Stage 2)

Starting with the second mission of GSLV Mk. III, the launcher will feature a Cryogenic Upper Stage carrying up to 28,000 Kilograms of Liquid Oxygen and Liquid Hydrogen. This upper stage is a larger version of the third stage of the Mk. II using a more powerful version of the Indian Cryogenic Engine. The upper stage measures about 13.55 meters in length and 4 meters in diameter with an inert mass on the order of five metric tons.

The third stage of the GSLV Mk. III is powered by a CE-20 cryogenic engine that is based on the CE-7.5 of the GSLV Mk. II, but delivers a much higher thrust of 200 Kilonewtons.

The engine is an open-cycle Gas Generator engine which uses a small fraction of the fuel and oxidizer flow from the turbopumps to burn it in a Gas Generator that delivers the high-pressure gas needed to drive the turbopumps of the fuel and oxidizer. The propellants are then injected into the combustion chamber where they are burned to generate thrust while the gas generator exhaust is ejected through a nozzle and not burned in the chamber.

The CE-20 engine operates at a fixed thrust level that can be set to 180 to 220 Kilonewtons. It operates at a mixture ratio of 5.05 and a nominal chamber pressure of 60 bar to achieve a specific impulse of 443 seconds.

The engine has a dry mass of about 588 Kilograms and uses an extended nozzle with an area ratio of 100 for optimized operation in vacuum. The engine provides re-start capability and operates up to 640 seconds in a nominal GSLV Mk. III flight.

Vehicle control during burns is provided by two vernier engines that can be swiveled in all directions to provide three-axis control. During Coast Phases, a cold gas reaction control system is used for vehicle stabilization and re-orientations. The spacecraft is separated by a tension release device mounted at the separation interface with the third stage.

The third stage of the launch vehicle also houses the flight computers and the inertial guidance platform of the GSLV. The control system was developed and built in India. GSLV uses a Redundant Strap Down Inertial Navigation System/Inertial Guidance System that is housed in the equipment bay of the third stage. The digital control system of the launcher uses closed-loop guidance throughout the flight to ensure accurate injections into the target orbit.

Also mounted on the third stage is the communications system of the launch vehicle consisting of an S-Band system for telemetry downlink and a C-Band transponder that allows radar tracking and preliminary orbit determination.

The communications link is also used for range safety / flight termination that uses a dedicated system that is located on all stages of the vehicle and features separate avionics.

 <center><img src="https://drive.google.com/uc?export=view&id=1iVWsjmUxVu4w-b4cPVEQZAobGoSoZnHu" alt="drawing" width="320"/></center>


## S200 Solid Roket Booster (GSLV Mk III Strapon boosters)

GSLV Mk. III uses two S200 solid-fueled boosters that are attached to the Core Stage and deliver thrust for the initial portion of the flight.

Also known as LSB (Large Solid Booster), each of the S200 boosters stands 26.2 meters tall and consists of three segments holding a total of 206,690 Kilograms of propellant.

The booster casing is 3.2 meters in diameter while the Flex Nozzle at the base of the booster is 3.27 meters in diameter.

S200 is the third largest Solid Rocket Booster used in any launch vehicle after the Space Shuttle and the Ariane 5 rocket. Like these two other vehicles, S200 uses a HTPB-based propellant (Hydroxyl-terminated Ploy-butadiene).

Major components of the S200 Solid Rocket Booster are the Flex Nozzle, the motor case, the propellant, igniter and nose cone.

The booster itself consists of three steel segments – the head-end segment contains 27,100kg of propellant, the middle segment carries 97,380kg and the Nozzle-end segment is loaded with 82,210kg of propellants.

S200 is ignited by a highly reliable pyrogen igniter that is initiated by a secondary add on pyrogen igniter which generates enough hot particles to travel down the length of the booster to ignite the propellant.

The igniter is based on that used on the S139 first stage of the PSLV and GSLV Mk. II launch vehicles.

It is triggered through a redundant safe and arm device that initiates explosive transfer assemblies.

The Flex nozzle at the base of the booster is divergent to direct the gas flow from the chamber.

S200 uses a nozzle with a throat diameter of 88.6 centimeters and an area ratio of 12.1. The Flex Nozzle assembly has a length of 3.58 meters consisting of a convergent subassembly, a flexseat subassembly, a submerged subassembly, a divergent fore-end subassembly and a divergent aft-end subassembly.

For vehicle control, the Flex Nozzle can be vectored using two electro-hydraulic servo actuators. This planar movement provides pitch and yaw control and, in combination with the other S200 booster, can be used for roll control as well.

The actuation system is housed in a flared base shroud installed on the booster that also builds the interface to the Mobile Launch Platform. The boosters use a dual redundant control unit with FDI logic.

The S200 boosters are designed to operate at a fixed pressure and thrust profile (M-type) with a maximum thrust of 5,151 Kilonewtons (sea level) and a average SL thrust of 3,578kN corresponding to chamber pressures of 56.9 and 39.9 bar.

S200 operates for 130 seconds and is separated from the launch vehicle at T+149 seconds using pyrotechnic separation devices and six small solid-fueled jettison motors located in the nose and aft segments of the boosters.


 <center><img src="https://drive.google.com/uc?export=view&id=1k0RLZV6RTFp3TUSm-jhkTope4AMaFOfV" alt="drawing" width="320"/></center>
 <center><img src="https://drive.google.com/uc?export=view&id=1K3Zaqe15umdrwZKsiLaK9xAUWQTAnAMC" alt="drawing" width="320"/></center>

<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>

# Approximate Staging Solution

The required Δv for an LEO orbit of 400km is found to be 9400 m/s, factoring in the losses incurred due to drag and gravity. The launch is assumed to be from SDSC-SHAR with a latitude of θ = 13.7259°N.
For best result and easterly launch is preferred due to the effect of the Earth's rotation reducing the Δv.
Velcity imparted due to the Earth's rotation:

v$_{surf}$ = $\frac{2 π r_{e} cos(θ)}{1day}$ = 450.57 m/s

Thus required Δv = 9400-450.57 = 8949.42 m/s
Based on the velocity requirements, a two stage rocket is chosen with booster stages operating in parallel with the first stage during liftoff. This is because using a two stage rocket appears to have the optimal liftoff mass for the given mission requirements for the available propulsion technology. Addition of more stages causes small improvement in payload capacity while the complexity and weight increase is significant espaecially due to inert component weight.

The engines are chosen as follows :

Stage 0 - Two S200 strapons that operate in tandem with Stage 1 till their separation. $I_{sp,0}$ = 227s. Existing configuration of S200 will be used with adjustment to propellant mass and burn time accordingly.

Justification: The stage is chosen as such due to the requirement of high thrust during liftoff. Thus, the engines with higher thrust rating is chosen. A parallel configuration is chosen as it leads to lesser gravity losses for the upper stages as the strapons are discareded upon burnout.

Stage 1 - Vikas LRE stage with $I_{sp,1}$ = 330s (modified for higher $I_{sp}$) to continue operation till burnout after booster jettison. Existing configuration of Vikas engine will be used with adjustment to propellant mass and burn time accordingly.

Justification: The Vikas stage is chosen as such due to the requirement of high thrust and high $I_{sp}$ for the climb phase. Thus, the engines with higher $I_{sp}$ than  is chosen. 

Stage 2 - C 25 CUS with $I_{sp, vac, 2}$ = 443s. Existing configuration of C 25 will be used with adjustment to propellant mass and burn time accordingly.

Justification: The C25 stage is chosen as such due to the requirement of high $I_{sp}$ for the terminal phase. to reduce weight. This leads to lower payload mass for lower stages thereby causing lesser gravity losses and lower liftoff weight.

Based on current technology, the Δv requirements is shared between the stages as follows:

First stage in parallel with two boosters Δv = 2700 m/s

First stage after booster separation Δv = 2700 m/s

Second stage after booster separation Δv = 3549.42 m/s

## Stage 2 solution

$m_{f2}$ = $m_{pl}$ + $m_{i2}$ = 15 + 5 = 20 tons

Using Tsiolkovsky equation, we get

3549.42 = - $I_{sp,vac,2} g_{LEO}ln(\frac{m_{f2}}{m_{o2}})$

Using $I_{sp,vac,2}$ = 443 and $g_{LEO}$ = 8.8 m/$s^{2}$, we get $\frac{m_{f2}}{m_{o2}}$ = 0.4023

Using $m_{f2}$ = 20 tons, we get $m_{o2}$ = 49.71042 tons

This gives $m_{p2}$ = 29.71 tons

## Stage 1 solution after booster separation

$m_{f1}$ = $m_{02}$ + $m_{i1}$ = 49.71 + 5.3 = 55.01 tons

Using Tsiolkovsky equation, we get

2700 = - $I_{sp,1} g_{e}ln(\frac{m_{f1}}{m_{o1}})$

Using $I_{sp,1}$ = 330 and $g_{e}$ = 9.8 m/$s^{2}$, we get $\frac{m_{f1}}{m_{o1}}$ = 0.4339

Using $m_{f1}$ = 55.01 tons, we get $m_{o1}$ = 126.773 tons

This gives $m_{p1}$ = 71.763 tons

## Stage 0 (2 boosters) and Stage 1 parallel operation solution 

$m_{f0}$ = $m_{01}$ + 2×$m_{i0}$ = 127.773 + 2×31.3 = 189.373 tons

Since here two different engines are oprating in parallel, the effective $I_{sp}$ can be computed as follows:

$I_{sp, eff}$ = $\frac{(2×I_{sp,0}×\dot{m}_{0} + I_{sp,1}×\dot{m}_{1})}{2×\dot{m}_{0} + \dot{m}_{1}}$ = 235.28 s

Using Tsiolkovsky equation, we get

2700 = - $I_{sp,eff} g_{e}ln(\frac{m_{f0}}{m_{o}})$

Using $I_{sp,1}$ = 330 and $g_{e}$ = 9.8 m/$s^{2}$, we get $\frac{m_{f1}}{m_{o1}}$ = 0.3100

Using $m_{f0}$ = 189.373 tons, we get $m_{o}$ = 611.61 tons

This gives $m_{p0}$ = 422.417 tons

But this mass is given the combined propellant mass of the two boosters and the portion of propellant consumed by stage 1 during the parallel oparation with the boosters.

Since the mass flow rate of both booster and first stage is available and assumimg constant burn profile, the burn time for parallel burn of boosters with stage 1 can be found as:

$t_{b,parallel}$ = $\frac{422.417×1000}{2×\dot{m}_{0} + \dot{m}_{1}}$ = 122.02 s

Using the burn time and the propellant flow rates of the two engines, we can calculate the propellant masses of the engines as shown:

Taking $\dot{m}_{0}$ = 1589.92 kg/s and $\dot{m}_{1}$ = 278.04 kg/s we get $m_{p, booster}$ = 194.195 tons each and $m_{p1, parallel}$ = 34.126 tons. 

Thus the total propellant mass of stage 1 is $m_{p1, total}$ = 34.126 + 71.763 = 105.889 tons

# Finalised Stage Information



In [55]:
import pandas as pd
import numpy as np

df = pd.DataFrame({'Stage': ['Stage 1','Stage 2', 'Booster (Stage 0)', 'Total']})
df = pd.concat([df, pd.DataFrame(data = [[105.889, 5.3, 49.71],
                                         [29.71, 5, 15],
                                         [194.195, 31.3, 126.773], 
                                         [523.989, 71.9, 15]] , 
                                 columns=('Propellant mass (T)',
                                          'Structural mass (T)', 
                                          'Payload mass (T)'))],
               axis=1)

df

Stage  Propellant mass (T)  Structural mass (T)  \
0            Stage 1              105.889                  5.3   
1            Stage 2               29.710                  5.0   
2  Booster (Stage 0)              194.195                 31.3   
3              Total              523.989                 71.9   

   Payload mass (T)  
0            49.710  
1            15.000  
2           126.773  
3            15.000


<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>

# Engine Parameters for Each Stage

## Vikas (Stage 1 Engine)

From earlier design, $m_{p2}$ = 105.889 tons and $\dot{m}$ = 278.04 kg/s

We get the burn time as follows

$t_{b}$ = $\frac{m_{p1}}{\dot{m}}$ = 380s

$I_{sp,1}$ = 330 s

Using $I_{sp}$ = $\frac{T}{\dot{m}g_{e}}$, we can compute the thrust 

T = 899.18 kN at sea level (using $g_{e}$ = 9.8 m/$s^{2}$)

Given that for $N_{2}O_{4}$ + UDMH propellant, the characteristic velocity 
c* = 1652 m/s and the chamber pressure is given to be $P_{c}$ 58.5 bar = 5.85 MPa, the Throat area $A_{t}$ is calculated as follows

c* = $\frac{P_{c}A_{t}}{\dot{m}}$

This gives $A_{t}$ = 0.0785 $m^{2}$

The effective exhaust velocity is calculated as follows using $I_{sp,1}$ and $g_{e}$

c = $I_{sp,1}$×$g_{e}$ 

which gives c = 3234 m/s

The chamber pressure is given to be $P_{c}$ 58.5 bar = 5.85 MPa

The thrust coefficient $c_{f}$ can thus be found out using Thrust, $P_{c}$ and $A_{t}$ as shown

$c_{f}$ = $\frac{T}{P_{c}A_{t}}$ = 1.958

## CE 20 (Stage 2 Engine) 

From earlier design, $m_{p2}$ = 29.710 tons

Taking a burn time $t_{b}$ of 800s as per similar engines, we get the propellant flow rate as follows

$\dot{m}$ = $\frac{m_{p2}}{t_{b}}$ = 39.77 kg/s

$I_{sp,2}$ = 443 s

Using $I_{sp}$ = $\frac{T}{\dot{m}g_{e}}$, we can compute the thrust

T = 155kN at vacuum (using $g_{e}$ = 8.8 m/$s^{2}$)

Given that for LOX + L$H_{2}$ propellant, the characteristic velocity 
c* = 2428 m/s and the chamber pressure is given to be $P_{c}$ 60 bar = 6 MPa, the throat area $A_{t}$ is calculated as follows

c* = $\frac{P_{c}A_{t}}{\dot{m}}$

This gives $A_{t}$ = 0.0161 $m^{2}$

The effective exhaust velocity is calculated as follows using $I_{sp,2}$ and $g_{e}$

c = $I_{sp,2}$×$g_{e}$ 

which gives c = 3898.4 m/s

The chamber pressure is given to be $P_{c}$ 60 bar = 6 MPa

The thrust coefficient $c_{f}$ can thus be found out using Thrust, $P_{c}$ and $A_{t}$ as shown

$c_{f}$ = $\frac{T}{P_{c}A_{t}}$ = 1.6045

## S200 LSB (Stage 0 Engine)

From earlier design, $m_{p0}$ = 194.195 tons

Taking a burn time $t_{b}$ of 122s as per similar engines, we get the propellant flow rate as follows

$\dot{m}$ = $\frac{m_{p2}}{t_{b}}$ = 1591.76 kg/s

$I_{sp,1}$ = 227 s

Using $I_{sp}$ = $\frac{T}{\dot{m}g_{e}}$, we can compute the thrust as 

T = 3613.3 kN at sea level (using $g_{e}$ = 9.8)

Given that for HTPB 68/12/20 (Ammonium perchlorate 84–68%, 12% polymer binder,and 4–20% aluminum propellant), the characteristic velocity c* = 1577 m/s and the chamber pressure is given to be $P_{c}$ 39.9 bar = 3.99 MPa, the throat area $A_{t}$ is calculated as follows

c* = $\frac{P_{c}A_{t}}{\dot{m}}$

This gives $A_{t}$ = 0.6062 $m^{2}$

The effective exhaust velocity is calculated as follows using $I_{sp,1}$ and $g_{e}$

c = $I_{sp,1}$×$g_{e}$ 

which gives c = 2224.6 m/s

The chamber pressure is given to be $P_{c}$ 39.9 bar = 3.99 MPa

The thrust coefficient $c_{f}$ can thus be found out using Thrust, $P_{c}$ and $A_{t}$ as shown

$c_{f}$ = $\frac{T}{P_{c}A_{t}}$ = 1.4793

The above results are tabulated as shown below

In [54]:

df2 = pd.DataFrame({'Stage': ['Stage 1 (Vikas)','Stage 2 (CE 20)', 
                              'Booster (S200 LSB)']})
df2 = pd.concat([df2, pd.DataFrame(data = [[330, 380, 3234, 278.04, 1652, 
                                            899.18, 0.0785, 5.85, 1.958],
                                           [443, 800, 3898.4, 39.77, 2428, 
                                            155, 0.0161, 6.0, 1.6045],
                                           [227, 122, 2224.6, 1591.76, 
                                            1577, 3613.3, 0.6062, 3.99, 1.4793],], 
                                 columns=('Isp (s)','Burn time (s)', 'c (m/s)', 
                                          'mass flow rate (kg/s)',
                                          'c* (m/s)', 'Thrust at sea level (kN)',
                                          'Throat area (m^2)', 
                                          'Chamber pressure (MPa)', 
                                          'Thrust coefficient'))], axis=1)

df2

Stage  Isp (s)  Burn time (s)  c (m/s)  mass flow rate (kg/s)  \
0     Stage 1 (Vikas)      330            380   3234.0                 278.04   
1     Stage 2 (CE 20)      443            800   3898.4                  39.77   
2  Booster (S200 LSB)      227            122   2224.6                1591.76   

   c* (m/s)  Thrust at sea level (kN)  Throat area (m^2)  \
0      1652                    899.18             0.0785   
1      2428                    155.00             0.0161   
2      1577                   3613.30             0.6062   

   Chamber pressure (MPa)  Thrust coefficient  
0                    5.85              1.9580  
1                    6.00              1.6045  
2                    3.99              1.4793


<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>


# Verification of Δv and Acceleration Constraints

The values of accelerations for each stage can be computed as follows

a(t) = $\frac{T - (m_{0}-\dot{m}t)gcos(ωt)}{m_{0}-\dot{m}t}$

## Stage 2 

$m_{0,2}$ = 49.71 tons

$\dot{m}_{2}$ = 0.03977 tons/s

T = 155 kN
 
g = 8.8 m/$s^{2}$ 

$a_{2}$(t) = $\frac{155 -(49.71-0.03977t)×8.8×cos(ωt)}{49.71-0.03977t}$

Assuming vehicle attitude to be almost horizontal during terminal phase and thus taking ωt = constant = 90° we get 

$a_{2}$(t) = $\frac{155}{49.71-0.03977t}$

Plotting for t = 0 s to 800 s (burn duration), we see that the maximum attained acceleration is given as

$η_{max, 2}$ = 8.662 m/$s^{2}$ = 0.9843 g

The plot of $a_{2}$ vs t is given below

![Vikas Engine](https://drive.google.com/uc?export=view&id=1u3VhOMDry4phZ1Qb_vh2cpfJQo6SZbc4)


## Stage 1 (after booster separation)

$m_{0,2}$ = 126.773 tons

$\dot{m}_{2}$ = 0.27804 tons/s

T = 899 kN
 
g = 9.8 m/$s^{2}$ 

$a_{1}$(t) = $\frac{899 -(126.773-0.27804t)×8.8×cos(ωt)}{126.773-0.27804t}$

Assuming vehicle attitude to change at a constant rate from 45° to 90° over the entire burn time duration,  we get ωt = 45° + $\frac{45°t}{258.1}$

$a_{1}$(t) = $\frac{899 -(126.773-0.27804t)×8.8×cos(45° + \frac{45°t}{258.1})}{126.773-0.27804t}$

Plotting for t = 0 s to 258.1 s (burn duration), we see that the maximum attained acceleration is given as

$η_{max, 1}$ = 16.331 m/$s^{2}$ = 1.6664 g

The plot of $a_{1}$ vs t is given below

![a2 vs t](https://drive.google.com/uc?export=view&id=1YM3TgYV0S9tcVYP_wm5T8Dv0PiOx40d7)

<br>
<br>
<br>
<br>
<br>
<br>

## Stage 0 (2 boosters) and Stage 1 parallel operation solution 

$m_{0,o}$ = 611.61 tons

$\dot{m}_{0}$ = 0.27804 + 2×1.5892 = 3.46156 tons/s

T = 899 + 2×3613.3 = 8125.5 kN
 
g = 9.8 m/$s^{2}$ 

$a_{0}$(t) = $\frac{8125.5 -(611.61-3.46156t)×8.8×cos(ωt)}{611.61-3.46156t}$

Assuming vehicle attitude to change at a constant rate from 0° to 45° over the entire burn time duration,  we get ωt = $\frac{45°t}{258.1}$

$a_{0}$(t) = $\frac{8125.5 -(611.61-3.46156t)×8.8×cos(\frac{45°t}{258.1})}{611.61-3.46156t}$

Plotting for t = 0 s to 122 s (burn duration), we see that the maximum attained acceleration is given as

$η_{max, 0}$ = 35.625 m/$s^{2}$ = 3.6352 g

The plot of $a_{0}$ vs t is given below

![a2 vs t](https://drive.google.com/uc?export=view&id=1fURwnfNbYz5c61cKVOaicycEXjcU1Tha)



<br>
<br>
<br>
<br>
<br>
<br>

# References and Sources

[1] *Rocket Propulsion Elements*, Ninth Edition, George P. Sutton, Oscar Biblarz, Wiley publications, 2017

[2] *Rocket Propulsion*, First Edition, Stephen D. Heister, William E. Anderson, Timothée Pourpoint, R. Joseph Cassady, Cambridge University Press, 2019

[3] *Lecture slides of AE333 Aerospace Propulsion*, V. R. Kowsik Bodi, IIT Bombay, 2022

[4] *Analysis of Thrust Coefficient in a Rocket Motor*, P. Bose, K.M Pandey, International Journal of Engineering and Advanced Technology (IJEAT), Volume-1 Issue-3, February 2012

[5] https://www.isro.gov.in/media_isro/pdf/GSLV%20MkIII-M1.pdf

[6] https://spaceflight101.com/members/wp-content/uploads/sites/54/2017/02/GSLV-MkIII.pdf

[7] https://en.wikipedia.org/wiki/Hydroxyl-terminated_polybutadiene

[8] https://spaceflight101.com/spacerockets/pslv/#google_vignette
